lib/hummingbird.ex

```diff
@@ -70,12 +70,16 @@ defmodule Hummingbird do
       data: %{
         conn: Helpers.sanitize(conn),
         caller: opts.caller,
-        trace_id: conn.assigns[:trace_id],
-        span_id: conn.assigns[:span_id],
-        parent_id: conn.assigns[:parent_id],
+        traceId: conn.assigns[:trace_id],
+        id: conn.assigns[:span_id],
+        parentId: conn.assigns[:parent_id],
         user_id: conn.assigns[:current_user][:user_id],
         route: conn.assigns[:request_path],
-        service_name: opts.service_name
+        serviceName: opts.service_name,
+        name: opts.service_name,
+        kind: "application"
```

Comment:
this is pretty debatable

I'd be ok with other similar stuff like `service` too. just something to denote that it's the code we wrote that's producing these spans

Reply:
Works for me.  Only thing I'm considering is namespacing, which I've seen discussed multiple times.

https://honeycombpollinators.slack.com/archives/C4T3A32CX/p1593786421378200 is a good example of the discussion.

Might hate this, but `app.user_id`, `app.route`, and.... `app.conn` for even more crappiness is something would like to discuss.

Think it's as simple and making `app: %{...}` for these.

Reply:
yeah it might make sense to make it configurable based on the plug invocation or something in the conn, so you could say `plug Hummingbird, kind: "app.initial_endpoint"` or "app.router" or "app.command_sender" or something

I'd like to separate out that configurability into a separate pr though so I don't have to change any consumers when publishing this update

do you think "app" or "application" @bjornrud ?